Dockerfile_cli

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=nansencenter/geospaas:2.0.0-slim
+ARG BASE_IMAGE=nansencenter/geospaas:2.3.3-slim
```

Comment:
you need to change the version in the .travis.yml file

Reply:
@aperrin66 Done!